README.md

```diff
@@ -138,9 +138,9 @@ For more instructions (including OS X) check out https://gist.github.com/wolfv/f
 
 ### Development installation
 
-Make sure to have the following requirements in your conda environment:
+Make sure you install mamba's requirements in your conda environment:
 
-`mamba install cmake compilers pybind11 libsolv libarchive libcurl nlohmann_json pip cpp-filesystem yaml-cpp reproc-cpp -c conda-forge`
+`mamba env update --name <conda_env> --file environment-dev.yml`
```

Comment:
I guess we could propose to use `install` command to be consistent with text `install mamba's requirements` ?
But also propose to `update` an existing env :)

Reply:
Sure! `conda install` doesn't like yaml files though, so I had to place the dependencies in a new text file. Thanks for the prompt answer :)